built_in_tasks/target_tracking_task.py

```diff
@@ -650,6 +655,7 @@ def _while_tracking_out(self):
 
         # Check if the trial is over and there are no more target frames to display
         if self.frame_index+self.lookahead >= np.shape(self.targs)[0]:
+            cursor_pos = [5., 5., -8.]
```

Comment:
I think we should remove this. Doesn't make sense for in booth/in cage experiments where cursor position is/could be be continuous. Wouldn't want the cursor to jump between trials

Reply:
Also the "starting_pos" trait doesn't seem to do anything. No matter what xyz position you specify, the cursor always appears at center of screen

Reply:
The starting_pos isn’t really useful when you’re doing position control but it is when you’re using velocity control.

Reply:
Got it